Shared/src/shared/models/card/Card.java

```diff
@@ -32,6 +33,7 @@ public Card(String name, //TODO refactor other constructors to use this one.
                 String description,
                 String spriteName,
                 CardType type,
+                boolean isCustom,
```

Comment:
I feel like there's probably a better way to tell if something is custom or not, but I can't think of it... This does work to make sure that people can put balance cards in `Custom_Cards` and still load the old sprites and stuff right?